content/md/posts/2023-07-08-unit-testing.md

```diff
@@ -0,0 +1,66 @@
+{:title "Unit testing"
+ :layout :post
+ :tags  ["cljs" "testing"]
+ :toc true}
+
+
+## Overview
+
+This article covers what unit testing is, why it is useful, and an example of unit testing in a project.
+
+## What is unit testing?
+
+ Unit testing is about testing a function, using the inputs and verifying the outputs. Unlike acceptance testing which tests the result of a collection of functions interacting together, unit testing just grabs one function and sees whether it reacts correctly given certain inputs.
```

Comment:
There's a space at the beginning... does markdown ignore spaces like that?
Also I think acceptance testing which `tests the application as a black box, from the user point of view`

Reply:
I did not realize that I was leaving spaces at the beginning. I'll go and remove those.

Reply:
I'll also update the description of acceptance testing so it defines it as testing the application from the user's perspective.